src/main/java/com/waverley/listeners/TestListener.java

```diff
@@ -0,0 +1,36 @@
+package com.waverley.listeners;
+
+import lombok.extern.slf4j.Slf4j;
+import org.testng.IInvokedMethod;
+import org.testng.IInvokedMethodListener;
+import org.testng.ITestResult;
+
+import java.time.LocalDate;
+
+/**
+ * Listeners test example.
+ */
+
+@Slf4j
+
```

Comment:
remove redundant lines to increase readability

Reply:
Fixed.